src/components/TargetsMap/Indicator/Indicator.scss

```
@@ -0,0 +1,48 @@
+@import 'styles/constants';
+
+.map__indicator {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+}
+
+.map__indicator__circle {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  transform: translate(0, -50%);
+  width: 2rem;
+}
```

You can write lines 3 - 15 in this way
```suggestion
.map__indicator {
  align-items: center;
  display: flex;
  flex-direction: column;
  
    &__circle {
      align-items: center;
      display: flex;
      justify-content: center;
      transform: translate(0, -50%);
      width: 2rem;
   }
}
```